<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1072</width>
    <height>629</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pushButton_12">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>540</y>
      <width>161</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">float: rigth;
margin-right: 3%;
margin-top: 5px;
border-radius: 5px;
-moz-border-radius: 5px;
-webkit-border-radius: 5px;
padding: 5px 9px;
font-size: 1.2em;
background-color: rgba(236, 101, 80, 1);
text-shadow: #4545545 0 0 2px;
border-bottom: 4px solid rgba(217, 91, 72, 1);
color: white;
font-family: 'Roboto Slab', serif;</string>
    </property>
    <property name="text">
     <string>Выход с программы</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1041</width>
      <height>291</height>
     </rect>
    </property>
    <column>
     <property name="text">
      <string>Даты</string>
     </property>
     <property name="font">
      <font>
       <bold>true</bold>
      </font>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Следующий день</string>
     </property>
     <property name="font">
      <font>
       <bold>true</bold>
      </font>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Предыдущий день</string>
     </property>
     <property name="font">
      <font>
       <bold>true</bold>
      </font>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Високосный год</string>
     </property>
     <property name="font">
      <font>
       <bold>true</bold>
      </font>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Номер недели</string>
     </property>
     <property name="font">
      <font>
       <bold>true</bold>
      </font>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Дней до дня рождения</string>
     </property>
     <property name="font">
      <font>
       <bold>true</bold>
      </font>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Разница со следующей датой</string>
     </property>
     <property name="font">
      <font>
       <bold>true</bold>
      </font>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Разница с текущей датой</string>
     </property>
     <property name="font">
      <font>
       <bold>true</bold>
      </font>
     </property>
    </column>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>510</y>
      <width>401</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:red;</string>
    </property>
    <property name="text">
     <string>ВСЕ ДАТЫ ИСПОЛЬЗУЮТСЯ В ФОРМАТЕ : ДД.ММ.ГГГГ</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_17">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>540</y>
      <width>171</width>
      <height>36</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">float: rigth;
margin-right: 3%;
margin-top: 5px;
border-radius: 5px;
-moz-border-radius: 5px;
-webkit-border-radius: 5px;
padding: 5px 9px;
font-size: 1.2em;
background-color: rgba(236, 101, 80, 1);
text-shadow: #4545545 0 0 2px;
border-bottom: 4px solid rgba(217, 91, 72, 1);
color: white;
font-family: 'Roboto Slab', serif;</string>
    </property>
    <property name="text">
     <string>Дней до дня рожения</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_6">
    <property name="geometry">
     <rect>
      <x>840</x>
      <y>510</y>
      <width>31</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>470</y>
      <width>341</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Написать слева день, справа месяц рождения</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_7">
    <property name="geometry">
     <rect>
      <x>870</x>
      <y>510</y>
      <width>31</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_8">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>340</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_18">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>380</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">float: rigth;
margin-right: 3%;
margin-top: 5px;
border-radius: 5px;
-moz-border-radius: 5px;
-webkit-border-radius: 5px;
padding: 5px 9px;
font-size: 1.2em;
background-color: rgba(236, 101, 80, 1);
text-shadow: #4545545 0 0 2px;
border-bottom: 4px solid rgba(217, 91, 72, 1);
color: white;
font-family: 'Roboto Slab', serif;</string>
    </property>
    <property name="text">
     <string>Добавить</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_9">
    <property name="geometry">
     <rect>
      <x>870</x>
      <y>300</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_10">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>300</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_21">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>340</y>
      <width>314</width>
      <height>36</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">float: rigth;
margin-right: 3%;
margin-top: 5px;
border-radius: 5px;
-moz-border-radius: 5px;
-webkit-border-radius: 5px;
padding: 5px 9px;
font-size: 1.2em;
background-color: rgba(236, 101, 80, 1);
text-shadow: #4545545 0 0 2px;
border-bottom: 4px solid rgba(217, 91, 72, 1);
color: white;
font-family: 'Roboto Slab', serif;</string>
    </property>
    <property name="text">
     <string>Изменить дату</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_12">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>300</y>
      <width>231</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Введите дату, чтобы добавить её</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_13">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>370</y>
      <width>391</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Чтобы изменить, введите слева номер, который хотите</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_22">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>540</y>
      <width>161</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">float: rigth;
margin-right: 3%;
margin-top: 5px;
border-radius: 5px;
-moz-border-radius: 5px;
-webkit-border-radius: 5px;
padding: 5px 9px;
font-size: 1.2em;
background-color: rgba(236, 101, 80, 1);
text-shadow: #4545545 0 0 2px;
border-bottom: 4px solid rgba(217, 91, 72, 1);
color: white;
font-family: 'Roboto Slab', serif;</string>
    </property>
    <property name="text">
     <string>Открыть файл</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>300</y>
      <width>316</width>
      <height>296</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QPushButton" name="pushButton_15">
       <property name="styleSheet">
        <string notr="true">float: rigth;
margin-right: 3%;
margin-top: 5px;
border-radius: 5px;
-moz-border-radius: 5px;
-webkit-border-radius: 5px;
padding: 5px 9px;
font-size: 1.2em;
background-color: rgba(236, 101, 80, 1);
text-shadow: #4545545 0 0 2px;
border-bottom: 4px solid rgba(217, 91, 72, 1);
color: white;
font-family: 'Roboto Slab', serif;</string>
       </property>
       <property name="text">
        <string>Cледующий день</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_14">
       <property name="styleSheet">
        <string notr="true">float: rigth;
margin-right: 3%;
margin-top: 5px;
border-radius: 5px;
-moz-border-radius: 5px;
-webkit-border-radius: 5px;
padding: 5px 9px;
font-size: 1.2em;
background-color: rgba(236, 101, 80, 1);
text-shadow: #4545545 0 0 2px;
border-bottom: 4px solid rgba(217, 91, 72, 1);
color: white;
font-family: 'Roboto Slab', serif;</string>
       </property>
       <property name="text">
        <string>Предыдущий</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_16">
       <property name="styleSheet">
        <string notr="true">float: rigth;
margin-right: 3%;
margin-top: 5px;
border-radius: 5px;
-moz-border-radius: 5px;
-webkit-border-radius: 5px;
padding: 5px 9px;
font-size: 1.2em;
background-color: rgba(236, 101, 80, 1);
text-shadow: #4545545 0 0 2px;
border-bottom: 4px solid rgba(217, 91, 72, 1);
color: white;
font-family: 'Roboto Slab', serif;</string>
       </property>
       <property name="text">
        <string>Високосный</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_13">
       <property name="styleSheet">
        <string notr="true">float: rigth;
margin-right: 3%;
margin-top: 5px;
border-radius: 5px;
-moz-border-radius: 5px;
-webkit-border-radius: 5px;
padding: 5px 9px;
font-size: 1.2em;
background-color: rgba(236, 101, 80, 1);
text-shadow: #4545545 0 0 2px;
border-bottom: 4px solid rgba(217, 91, 72, 1);
color: white;
font-family: 'Roboto Slab', serif;</string>
       </property>
       <property name="text">
        <string>Неделя</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_20">
       <property name="styleSheet">
        <string notr="true">float: rigth;
margin-right: 3%;
margin-top: 5px;
border-radius: 5px;
-moz-border-radius: 5px;
-webkit-border-radius: 5px;
padding: 5px 9px;
font-size: 1.2em;
background-color: rgba(236, 101, 80, 1);
text-shadow: #4545545 0 0 2px;
border-bottom: 4px solid rgba(217, 91, 72, 1);
color: white;
font-family: 'Roboto Slab', serif;</string>
       </property>
       <property name="text">
        <string>Между первой датой и следующей в файле</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_19">
       <property name="styleSheet">
        <string notr="true">float: rigth;
margin-right: 3%;
margin-top: 5px;
border-radius: 5px;
-moz-border-radius: 5px;
-webkit-border-radius: 5px;
padding: 5px 9px;
font-size: 1.2em;
background-color: rgba(236, 101, 80, 1);
text-shadow: #4545545 0 0 2px;
border-bottom: 4px solid rgba(217, 91, 72, 1);
color: white;
font-family: 'Roboto Slab', serif;</string>
       </property>
       <property name="text">
        <string>С текущей даты</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="label_14">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>400</y>
      <width>311</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>изменить, а справа дату, на которую хотите</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1072</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
